### DISCUSSION TITLE
1.13.0: The Arm Update

### DISCUSSION THREAD

**Discussion by Draylar** [Draylar/identity#189] [General]
1.13.0 is here, and it brings an exciting update: entity arms! You will no longer see the player's hand when you are disguised as a pig... instead, most entities have specifically tweaked hand models that will display instead. This update also contains several bug fixes, and sk_sk translations by SmajloSlovakian.

<hr /><em>This discussion was created from the release <a href='https://github.com/Draylar/identity/releases/tag/1.13.0-1.16.5'>1.13.0: The Arm Update</a>.</em>

**Comment by Ch1216** [comment]
Is this gonna be one of the last update since is listed as a "Release"

**Reply by MimzyFang** [reply]
no if this was the case we would still play Minecraft 1.0 as the newest version of the game. "Release" just means that the mod is in am more stable form now and no longer an experiment there are still things to add and Minecraft versions to support.

**Comment by Sage456123** [comment]
Which fabric api for this release?

**Reply by Draylar** [reply]
Latest Fabric API (>=34) will work, but I'm guessing ~>=31 will work as well.

**Reply by Sage456123** [reply]
okay but why are you ignoring my messages?

**Reply by MimzyFang** [reply]
@Sage456123 because you are impatient and annoying
